src/npctrade.cpp

```diff
@@ -202,6 +203,14 @@ int npc_trading::trading_price( Character const &buyer, Character const &seller,
                 return VisitResponse::SKIP;
             }
         }
+        // Prevent free items
```

Comment:
This resets the price of active bombs to 1 cent. 

I don't think it's a good idea to hardcode minimum values like this. For example thread is defined in JSON to have a price of 25 cents per 200 units (or unit price of 0.125 cents) and you're just introducing confusing by overriding the JSON definition.

Reply:
I didnt realise that was doing it to bombs, good spotting.  However with this PR, traders refuse to buy items that cost 1 cent - and active bombs will be on that category.  Perhaps it can be given another flag if this is not enough?

However thread having 25 cents per 200 units is setting it to below 1 cent per item, and that is the problem - I think there is no way to make this system sell something to you for less than a cent.  So setting it to 1 cent per item is the only way to stop you from buying it for free.

If you have a better suggestion for preventing items being sold for free, please let me know..  Otherwise I think 1c per item minimum is the best solution for now.